src/utils/cryptoModule.js

```diff
@@ -62,6 +63,10 @@ class Crypto {
 	getKeys(passphrase) {
 		return this.liskCrypto.getKeys(passphrase);
 	}
+
+	getAddressFromPublicKey(publicKey) {
+		return this.liskCrypto.getAddressFromPublicKey(publicKey);
```

Comment:
This should process the result from `lisk-js` into an object like `decryptMessage` does.